skyportal/handlers/api/source.py

```diff
@@ -131,6 +131,13 @@ def get(self, obj_id=None):
             description: |
               Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
               last_detected <= endDate
+          - in: query
+            name: group_id
```

Comment:
This should be a list, so that one can filter on any number of groups

Reply:
ok, done